### PR TITLE
urlButtons: Allow icon name to differ from icon description

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: src/libsync/theme.cpp [hunk 1]
```diff
@@ -688,7 +688,7 @@ bool Theme::wizardEnableWebfinger() const
     return false;
 }
 
-QVector<QPair<QString, QUrl>> Theme::urlButtons() const
+QVector<std::tuple<QString, QString, QUrl>> Theme::urlButtons() const
```
Comment (fmoc) [inline src/libsync/theme.cpp:691]: How about adding an actual class to house the data? This should make the code easier to read and understand.

Comment (TheOneRing) [reply src/libsync/theme.cpp:691]: Its use is currently very limited, adding yet another class is currently probably not worth it.

Comment (TheOneRing) [reply src/libsync/theme.cpp:691]: Hmm I could directly return an icon

Comment (TheOneRing) [reply src/libsync/theme.cpp:691]: Ah I can't as this code is generated ....

Comment (fmoc) [reply src/libsync/theme.cpp:691]: Good enough for me.

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-oc10</strong> https://drone.owncloud.com/owncloud/client/14754/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/14754/oc10/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/14754/oc10/guiReportUpload/serverlog.log

Screenshots:
- [simple_sharing_of_file_and_folder_by_public_link_without_password.png](https://cache.owncloud.com/public/owncloud/client/14754/oc10/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_without_password.png)

</details>

**Pull request merged**